airbyte-config/models/src/main/java/io/airbyte/config/EnvConfigs.java

```diff
@@ -47,6 +47,13 @@
   public static final String DATABASE_USER = "DATABASE_USER";
   public static final String DATABASE_PASSWORD = "DATABASE_PASSWORD";
   public static final String DATABASE_URL = "DATABASE_URL";
+  private static final String MINIMUM_WORKSPACE_RETENTION_DAYS = "MINIMUM_WORKSPACE_RETENTION_DAYS";
+  private static final String MAXIMUM_WORKSPACE_RETENTION_DAYS = "MAXIMUM_WORKSPACE_RETENTION_DAYS";
+  private static final String MAXIMUM_WORKSPACE_SIZE_MB = "MAXIMUM_WORKSPACE_SIZE_MB";
+
+  private static final long DEFAULT_MINIMUM_WORKSPACE_RETENTION_DAYS = 1;
+  private static final long DEFAULT_MAXIMUM_WORKSPACE_RETENTION_DAYS = 60;
+  private static final long DEFAULT_MAXIMUM_WORKSPACE_SIZE_MB = 5000;
```

Comment:
5g feels like a lot, but I'm not sure that I have a great heuristic for choosing a number.

Reply:
🤷‍♀️  seems fine to me, but i similarly don't have a great heuristic.